packages/driver/src/multi-domain/domain_fn.ts

```diff
@@ -13,23 +14,57 @@ interface RunDomainFnOptions {
   state: {}
 }
 
+interface serializedRunnable {
+  id: string
+  type: string
+  title: string
+  parent: serializedRunnable
+  ctx: {}
+  _timeout: number
+  titlePath: string
+}
+
+const rehydrateRunnable = (data: serializedRunnable): Runnable|Test => {
+  let runnable
+
+  if (data.type === 'test') {
+    runnable = Cypress.mocha.createTest(data.title, () => {})
+  } else {
+    runnable = new Cypress.mocha._mocha.Mocha.Runnable(data.title)
```

Comment:
This isn't _exactly_ accurate, runnable is the super class of what typically parents are, suites etc. This seems good enough for now.